server/services/curiousService.test.ts

```diff
@@ -168,224 +161,29 @@ describe('curiousService', () => {
   describe('getPrisonerInPrisonCourses', () => {
     it('should get In Prison Courses', async () => {
       // Given
-      const learnerEducationPage1Of1: LearnerEducationPagedResponse = learnerEducationPagedResponse(prisonNumber)
```

Comment:
The new C2 /learnerQualifications API is no longer a paged API, so the service method implementation and tests are much much simpler (ie. no need for a while loop to keep calling the API endpoint until we get `last:true` etc)